ispdb/jet.ne.jp.xml

```diff
@@ -5,25 +5,26 @@
     <displayName>JETINTERNET</displayName>
     <displayShortName>JET</displayShortName>
     <incomingServer type="pop3">
-      <hostname>pop.jet.ne.jp</hostname>
-      <port>110</port>
-      <socketType>plain</socketType>
+      <hostname>pop02.jet.ne.jp</hostname>
+      <port>995</port>
+      <socketType>SSL</socketType>
       <username>%EMAILLOCALPART%</username>
-      <authentication>password-encrypted</authentication>
+      <authentication>password-cleartext</authentication>
     </incomingServer>
     <incomingServer type="imap">
-      <hostname>imap.jet.ne.jp</hostname>
+      <hostname>imap02.jet.ne.jp</hostname>
```

Comment:
This change is a regression from a generic hostname to a specific host. If we can avoid that, that would be good. Does the `imap.jet.ne.jp` hostname still work? If so, we should keep it.

(Settings published on the ISP's help pages are surprisingly often sub-optimal. We do not follow them, if there is a better config that works.)

Reply:
Changed in 3dffe857ffd92e40552dd88a23d34e91a17ed99e